tendrl/commons/tests/objects/global_details/test_global_details.py

```diff
@@ -0,0 +1,97 @@
+import __builtin__
+import maps
+from mock import patch
+import uuid
+
+from tendrl.commons.objects.cluster_tendrl_context import ClusterTendrlContext
+from tendrl.commons.objects.global_details import GlobalDetails
+from tendrl.commons.utils import etcd_utils
+
+
+# tests __init__ and render
+def test_render():
+    setattr(__builtin__, "NS", maps.NamedDict())
+    NS.tendrl_context = maps.NamedDict()
+    NS.tendrl_context.integration_id = "test_integration_id"
+
+    # TODO($username) Without the following two lines, this tests somehow
```

Comment:
you need replace $username with your github id

check: https://github.com/Tendrl/commons/pull/999/files#diff-271d62125a5b1fc0bd2350a19ea05cd5R255